bin/opcodes.mts

```diff
@@ -16,11 +16,11 @@ const ALL: (string | null)[] = [
 const TYPES = ['interface', 'code', 'debug', 'all'] as const;
 
 if (emitter.options.help) {
-  throw usage();
+  usage();
```

Comment:
An old version of TS didn't properly handle functions returning `never`, and we worked around that
by throwing the `never`. But now that's fixed and it's a lint error to throw `never`, so there are a
number of places where this is fixed.